lib/conf.d/bases_structure.xml

```diff
@@ -2580,7 +2580,7 @@
           </field>
           <field>
             <name>action</name>
-            <type>enum('push','add','validate','edit','collection','status','print','substit','publish','download','mail','ftp','delete','')</type>
+            <type>enum('push','add','validate','edit','collection','status','print','substit','publish','download','mail','ftp','delete', 'subdefCreation', '')</type>
```

Comment:
please also create a lib/classes/patch script, because this xml may become obsolete (due to syntax differences between mariadb, mysql5, mysql8 etc)

Reply:
ok